truss/patch/local_truss_patch_applier.py

```diff
@@ -28,6 +25,10 @@ def __init__(self, truss_dir: Path, env_exe, logger: logging.Logger) -> None:
         self._logger = logger
 
     def __call__(self, patches: List[Patch]):
+        from truss.templates.control.control.helpers.truss_patch.model_code_patch_applier import (
```

Comment:
why move the import?

Reply:
was causing some issues in `model_container_patch_applier.py` but that may have been related to #397, let me check once more if this is still necessary

Reply:
import at the top level looks to work fine, let me move this back